src/main/java/ome/services/RawPixelsBean.java

```diff
@@ -682,9 +687,23 @@ public synchronized Map<Integer, int[]> getHistogram(int[] channels,
             int binCount, boolean globalRange, PlaneDef plane) {
         errorIfNotLoaded();
 
-        if (requiresPixelsPyramid())
-            throw new ApiUsageException(
-                    "This method can not handle tiled images yet.");
+        //Find resolution level closest to max plane size without
+        //exceeding it
+        int resolutionLevel = -1;
+        for (int i = 0; i < buffer.getResolutionLevels(); i++) {
+            buffer.setResolutionLevel(i);
```

Comment:
Unfortunately this call throws for images created via the RawPixelService API as `RomioPixelBuffer` does not allow to set resolution levels - see e.g. https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/lastCompletedBuild/testReport/OmeroWeb.test.integration.test_histogram/TestHistogram/test_histogram_bin_count_10_/